src/vs/workbench/browser/parts/editor/editorStatus.ts

```diff
@@ -319,6 +319,7 @@ export class EditorStatus extends Disposable implements IWorkbenchContribution {
 		this._register(this.textFileService.untitled.onDidChangeEncoding(model => this.onResourceEncodingChange(model.resource)));
 		this._register(this.textFileService.files.onDidChangeEncoding(model => this.onResourceEncodingChange((model.resource))));
 		this._register(TabFocus.onDidChangeTabFocus(() => this.onTabFocusModeChange()));
+		this.onTabFocusModeChange();
```

Comment:
The `runAndSubscribe` helper combines these 2 lines for us:

https://github.com/microsoft/vscode/blob/2298cde055800679384713da7fd4da48037ebd32/src/vs/base/common/event.ts#L483-L486